app/components/dashboard/diabetes/bs_below200_graph_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <div id="bs-below-200" class="d-lg-flex w-lg-50 pr-lg-2">
   <%= render Dashboard::Card::GraphComponent.new(id: "bsBelow200PatientsTrend", data: graph_data, period: period) do |c| %>
-    <%= c.title(title: "Blood sugar &lt;200".html_safe, subtitle: t("bs_below_200_copy.reports_card_subtitle", region_name: region.name)) do |title| %>
+    <%= c.title(title: t("bs_below_200_copy.reports_card_title").html_safe, subtitle: t("bs_below_200_copy.reports_card_subtitle", region_name: region.name)) do |title| %>
```

Comment:
Is the copy specific for SL? Else, we can tie this in with the same Flipper flag

Reply:
the key "bs_below_200_copy" is already present in the default en.yml file. This text was translated though, here I have just added translation for it and added the updated needed for SL in en-LK.yml
This locale will be handled through DEFAULT_PREFERRED_DASHBOARD_LOCALE variable